src/Temporalio/Bridge/Cargo.toml

```diff
@@ -15,6 +15,11 @@ log = "0.4"
 parking_lot = "0.12"
 prost = "0.11"
 prost-types = "0.11"
+# We rely on Cargo semver rules not updating a 0.x to 0.y. Per the rand
+# documentation, before 1.0, minor 0.x updates _can_ break portability which can
+# cause non-determinism.
+rand = "0.8.5"
+rand_pcg = "0.3.1"
```

Comment:
This shouldn't be an issue since you're explicitly choosing the algorithm, I believe.

Reply:
You're probably right, but I was just scared knowing some of the higher level traits are helpers on the lower level and they call out multiple times that the library in general is only _portable_ for the same 0.x series. So I figured I'd call it out here.